### DISCUSSION TITLE
Is there a way to manage guild applications?

### DISCUSSION THREAD

**Discussion by uhAlexz** [Rapptz/discord.py#9937] [Q&A] [answered]
Hello! I was wondering if there is a way to manage guild applications with my discord.py bot. I've been wanting one of these bots to accept certain members into my guild.

**Comment by codeofandrin** [comment]
What excatly do you mean by "guild applications"? Do you mean applications/verification for guild servers (clans)?

**Reply by uhAlexz** [reply]
Yep!

**Reply by codeofandrin** [reply, marked as answer]
Clans are partially released at the moment and not documented in the API documentation on Discord's side, as you can see in this PR: https://github.com/discord/discord-api-docs/pull/6836

Because of this, this feature won't be implemented in the library at the moment either. But there's already an open PR: https://github.com/Rapptz/discord.py/pull/9899

I hope that answers your question.
